ext/standard/tests/general_functions/phpinfo-header-anchors.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+phpinfo() with clickable anchor tags
+--POST--
+a=b
```

Comment:
I don't think this is actually required.

Reply:
That was to trigger the HTML output of `phpinfo()`, which returns non-HTML on the default CLI runtime. I tried adding `--CGI--` to require CGI, but didn't seem to work. `--POST--` did the trick.

Reply:
I see, thanks for the explanation.

Reply:
Please leave a note about this in the test -- we'll otherwise lose this info, I'm sure of it ^_^